css/free5e.scss

```diff
@@ -27,11 +27,19 @@ table {
   width: 100%;
 }
 
-.spell {
+.spell,.monster,.item {
   font-style: italic;
   color: #152347;
+}
 
-  &:after {
-    content: '\00a0🪄';
-  }
+.spell:after {
+  content: '\00a0🪄';
+}
+
+.monster:after {
+  content: '\00a0🧚';
```

Comment:
This adds a little fairy symbol to monster names in the HTML output, similar to what I did for spells:

<img width="1037" height="396" alt="Monsters now have a small fairy symbol" src="https://github.com/user-attachments/assets/8185fd43-e625-41dc-8d77-aa74e731d154" />